infra/Makefile.toml

```diff
@@ -60,7 +60,7 @@ ACTORS_BUNDLE = "${CARGO_MAKE_WORKSPACE_WORKING_DIRECTORY}/../builtin-actors/out
 # If this wasn't present, any wait task is skipped.
 CARGO_MAKE_WAIT_MILLISECONDS = 5000
 # This wait time seems to work locally.
-CMT_WAIT_MILLIS = 10000
+CMT_WAIT_MILLIS = 30000
```

Comment:
Wow that's long! 

https://github.com/consensus-shipyard/fendermint/pull/344 should help